CodeGeneration/Sources/generate-swiftsyntaxbuilder/ResultBuildersFile.swift

```diff
@@ -27,7 +27,7 @@ let resultBuildersFile = SourceFile {
     let expressionType: Type = (node.collectionElementChoices?.isEmpty ?? true) ? elementType.parameterType : Type(MemberTypeIdentifier("\(type.buildable).Element"))
 
     StructDecl(
-      attributes: [CustomAttribute(trailingTrivia: .newline, attributeName: Type("resultBuilder"))],
+      attributes: [.customAttribute(CustomAttribute(trailingTrivia: .newline, attributeName: Type("resultBuilder")))],
       modifiers: [DeclModifier(name: .public)],
       identifier: "\(type.syntaxKind)Builder") {
```

Comment:
Any reason not to use the parsing initializer here too, i.e. something along the lines of

```swift
StructDecl("@resultBuilder public struct \(type.syntaxKind)Builder") {
  ...
}
```

Reply:
Thanks, I just missed that.